File: Android/app/src/main/java/com/hhp227/datemate/ui/auth/forgotpassword/ForgotPasswordUiState.kt
package com.hhp227.datemate.ui.auth.forgotpassword

data class ForgotPasswordUiState(
    val email: String = "",
    val emailError: String? = null,
    val isLoading: Boolean = false,
    val isEmailSent: Boolean = false,
    val message: String? = null
)

File: iOS/DateMate/UI/Auth/ForgotPassword/ForgotPasswordUiState.swift
//
//  ForgotPasswordUiState.swift
//  DateMate
//
//  Created by 홍희표 on 2025/11/27.
//

import Foundation

struct ForgotPasswordUiState {
    var email: String = ""
    var emailError: String = ""
    var isLoading: Bool = false
    var isEmailSent: Bool = false
    var message: String? = nil
}
